### PR TITLE
:star: Discord Templates presence!

### PR DESCRIPTION
**Pull request by eggsywashere**
🔗 **Partner Presence!**

Discord Server Templates is where you can easily find templates for your Discord server. Templates are free and everyone can easily add their own templates! You can also get upvotes for your templates and become the **most** votes templates and get on our [top templates](https://discordtemplates.com/templates/top) page! Visit our website [here](https://discordtemplates.com).

Here are some examples:
image

image

[Twitter](https://twitter.com/dctemplates) - [Team](https://discordtemplates.com/about#team) - [Discord Server](https://discordtemplates.com/redirect/discord)

🚧 PS: Our website is still in development/public testing. We've tried our best to outcome with a stable website but expect some bugs, issues and problems at this point. We'll try our best to provide you a stable website.

### CODE REVIEW

File: websites/D/Discord Templates/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  },
+  "url": "discordtemplates.com",
+  "description": {
+    "en": "Start creating your own community with our awesome templates. Build your own kingdom.",
+    "tr": "Sitemizdeki ücretsiz şablonlarla kendi topluluğunuzu kurun ve eğlenin!"
+  },
+  "service": "Discord Templates",
+  "version": "1.0.0",
+  "logo": "https://discordtemplates.com/assets/images/logo-no-particles.png",
+  "thumbnail": "https://i.imgur.com/QU2NHDW.png",
```
Comment (Bas950) [inline websites/D/Discord Templates/dist/metadata.json:15]: upload logo to imgur

**Comment by eggsywashere** [comment]
> Partner but no button: false?

sorry, first partner sir :)

**Comment by Bas950** [comment]
> > Partner but no button: false?
> 
> sorry, first partner sir :)

Double check if it still works tho, cause they need to add meta tag

**Comment by eggsywashere** [comment]
> Double check if it still works tho, cause they need to add meta tag

OH, sure, thanks

**Comment by snowy1000** [comment]
^^

**Comment by cembkoztekin** [comment]
👌

**Pull request merged**